content/en/project/_index.md

```diff
@@ -1,5 +1,11 @@
 
-## Project overview
+# What is the Medley Interlisp Project?
+
+The Medley Interlisp Project was created to revive Medley Interlisp. 
+
+# [What does Interlisp have to do with current systems?](about)
+What are the main features that make it interesting to revive?
+How might we apply the lessons from Interlisp to current systems?
 
```

Comment:
is this a repeat of other sections?

Reply:
Might it be worth (if it hasn't been done) creating a tree diagram that builds from the initial root index to the leaves showing where content will exist?   Line 6 looks like a link back to the about section.  In my mental model there should be a hierarchy that forms the overall layout of information.  You start at the root and dig deeper into content to gain more insight on specific areas.  I'm sure there be places were it makes sense to hyperlink across nodes, but the general flow should be root to leaves.